index.html

```diff
@@ -357,25 +357,19 @@ <h2>
             </div>
     
             
-            <form>
+            <form action="https://formspree.io/f/xnqrgeew" method="POST"> 
```

Comment:
- [ ] I recommend fix the margin/padding here to be the same as the figma design so please fix it💯🚀🔥

- [ ] I sugest adding more font weight on the heading to make it the same as figma template 😊💯🔥💯
- your design

![image](https://user-images.githubusercontent.com/58553711/199228663-a21c90d3-fd2d-405e-9ffd-cfbf4c674c44.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/199228745-dd2e7281-aef3-4aad-a58e-66533e175685.png)

Reply:
## resoved 

- changed the background, I downloaded the wrong one 
- imported a font-weight of 700 to fix the issue
- properly used the margin and padding to fix the misalignment 

Thank you for noticing that error